app/src/main/java/com/andela/javadevelopers/view/DetailActivity.java

```diff
@@ -32,6 +40,31 @@ protected void onCreate(Bundle savedInstanceState) {
         showProfile();
 
         getSupportActionBar().setTitle(userName);
+
+    }
+
+    /**
+     * @return the IntentBuilder
+     */
+    private Intent createShareIntent() {
+
```

Comment:
@Abudu-Samuel 
Please get rid of this white space

Reply:
Done